cli_reference/cli_by_example_content.adoc

```diff
@@ -251,14 +251,17 @@
 
 [options="nowrap"]
 ----
-  // Tag the current image for the image stream 'openshift/ruby' and tag '2.0' into the image stream 'yourproject/ruby with tag 'tip':
+  // Tag the current image for the image stream 'openshift/ruby' and tag '2.0' into the image stream 'yourproject/ruby with tag 'tip'.
```

Comment:
Please leave the colons the way there were. That's the right way to do it, explain and then show the actual command (applies to all changes in here).